backend/config/curation-api.yml

```diff
@@ -349,57 +346,41 @@ paths:
           $ref: "#/components/responses/400"
         "404":
           $ref: "#/components/responses/404"
-    patch:
-      summary: Update a Dataset's curator tag.
-      tags:
-        - collection
+    post:
+      summary: Create a Dataset ID to be used in a future upload.
       description: |
-        Update a Dataset's curator-provided tag. The Dataset MUST belong to a either a private Collection OR a Revision. 
-        MUST indicate the Dataset by providing *one* of either the `curator_tag` (non-null) OR the `dataset_id` 
-        parameters.
-
-        * Required: provide your access token in the request header as `Authentication: bearer <access_token>`.
+        Create an new empty Dataset and return the `dataset_id`. The `dataset_id` is used when uploading a Dataset file 
+        to S3 to identify what Dataset the file belongs to.
+      operationId: backend.corpora.lambdas.api.v1.curation.collections.collection_id.datasets.actions.post
       security:
         - curatorAccess: []
-      operationId: backend.corpora.lambdas.api.v1.curation.collections.collection_id.datasets.actions.patch
+      tags:
+        - collection
       parameters:
         - $ref: "#/components/parameters/path_collection_id"
-        - $ref: "#/components/parameters/query_curator_tag"
-        - $ref: "#/components/parameters/query_dataset_id"
-      requestBody:
-        content:
-          application/json:
-            schema:
-              type: object
-              additionalProperties: false
-              properties:
-                curator_tag:
-                  $ref: "#/components/schemas/curator_tag"
-            example:
-              curator_tag: "new/curator_tag"
       responses:
-        "204":
-          $ref: "#/components/responses/204"
-        "400":
-          $ref: "#/components/responses/400_multiple"
+        "201":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  dataset_id:
+                    $ref: "#/components/schemas/dataset_id"
```

Comment:
This should be `id`, as it is with the subsequent endpoint (PUT) for this route.